source/dubregistry/mongodb.d

```diff
@@ -11,22 +11,39 @@ private Nullable!MongoClientSettings _mongoSettings;
 
 @safe:
 
-MongoClientSettings mongoSettings() {
+/**
+Initializes mongoSettings if it's null
+
+Params:
+    anonymousAuth = whether to allow anonymous access
+
+Returns: MongoClientSettings
+*/
+MongoClientSettings mongoSettings(bool anonymousAuth = false) {
 	if (_mongoSettings.isNull)
 	{
 		import std.process : environment;
 		auto mongodbURI = environment.get("MONGODB_URI", environment.get("MONGO_URI", "mongodb://127.0.0.1"));
 		logInfo("Found mongodbURI: %s", mongodbURI);
 		_mongoSettings = MongoClientSettings.init;
 		parseMongoDBUrl(_mongoSettings, mongodbURI);
-		_mongoSettings.authMechanism = MongoAuthMechanism.scramSHA1;
+
+		if (!anonymousAuth)
+		{
+			import std.stdio;
+			writeln("Using scramSHA1");
```

Comment:
Use `logInfo`

Reply:
We don't even need to log this, I'll remove it